Samples/EventHorizon/Server2/Server.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../../default.props"/>
+
+  <PropertyGroup>
+    <AssemblyName>Dolittle.Runtime.Server</AssemblyName>
```

Comment:
Name of assembly. Just call it Server2, as the sample - it would be good to have a better name than "Server" and "Server2"